test/runtime/samples/binding-select-multiple/_config.js

```diff
@@ -4,16 +4,6 @@ export default {
 		selected: [ 'two', 'three' ]
 	},
 
-	ssrHtml: `
```

Comment:
Possibly I'm misunderstanding how these tests work, but I think I'd expect this to look something like the following:
```
	ssrHtml: `
		<select multiple>
			<option value="one">one</option>
			<option value="two" selected="selected">two</option>
			<option value="three" selected="selected">three</option>
		</select>
		<p>selected: two, three</p>
	`,
```

Reply:
it's over here: 

https://github.com/sveltejs/svelte/blob/b662c7fd41b3b778ea1ae80b0a4d86c239ffcfe7/test/server-side-rendering/index.ts#L207-L211

for server_side_rendering test, it'll check the html content with `ssrHtml || html` if exists.
since the changes now make the `ssrHtml` the same as `html`, i'm removing the `ssrHtml`